src/chunks/metadata.rs

```diff
@@ -16,18 +15,18 @@ use crate::io::ReadMetadata;
 /// finalfusion metadata in TOML format.
 #[derive(Clone, Debug, PartialEq)]
 pub struct Metadata {
-    inner: Value,
+    inner: toml::Table,
```

Comment:
Any reason not to import `toml::Table` above?

Reply:
Changed it to import the type.